hotel/models.py

```diff
@@ -134,4 +134,4 @@ def check_out_date(self):
 
 class RoomAssignment(MagModel):
     room_id     = Column(UUID, ForeignKey('room.id'))
-    attendee_id = Column(UUID, ForeignKey('attendee.id'), unique=True)
+    attendee_id = Column(UUID, ForeignKey('attendee.id'))
```

Comment:
I had to run this command locally and I'll run it on the staging and production server before deploying:
```sql
alter table room_assignment drop constraint room_assignment_attendee_id_key;
```